src/main/java/com/fidesmo/fdsm/CommandLineInterface.java

```diff
@@ -41,6 +41,7 @@
     final static String OPT_DELETE_APPLET = "delete-applet";
     final static String OPT_CARD_APPS = "card-apps";
     final static String OPT_CARD_INFO = "card-info";
+    final static String OPT_DEVICE_INFO = "device-info";
```

Comment:
The option is confusing, IMHO, what's the difference between `card-info` and `device-info`? Maybe we can left only one of them?

Reply:
card-info is offline-capable function, whereas device-info triggers online systems and databases.

Reply:
Don't see a reason why to not merge them? If there's no connection – just omit missing information, we did the same in other places

Reply:
I can do that as well, for avoiding adding new options